chanfitness/chaneventstore.go

```diff
@@ -142,6 +146,10 @@ func NewChannelEventStore(config *Config) *ChannelEventStore {
 // information from the store. If this function fails, it cancels its existing
 // subscriptions and returns an error.
 func (c *ChannelEventStore) Start() error {
+	if !atomic.CompareAndSwapUint32(&c.started, 0, 1) {
+		return nil
```

Comment:
Let's always return an error when `Start` is called twice?

Reply:
Would also put the log `log.Info("ChannelEventStore starting")` before this check

Reply:
hmm I think the current design calls them twice, erroring out might cause problems, let's see what the itests say with the new design.